refractored commands to use subprocess

alisonqiu opened the pull request.

alisonqiu committed:
refractored commands to use subprocess

shreyasminocha committed:
Use given path to implementation

alisonqiu committed:
Merge pull request #1 from shreyasminocha/use-second-arg

Use given path to implementation

zawie commented:
Thanks for making a pull request 😁

What exactly does this do?

Seems like it's changing some paths and allowing users to specify implementation path.

If you clean up commented out code  and leave any needed instructions for future 412 students I'll merge it in 🚀🚀

Also trusting that it works bc I can't  log into clear to test. Can you double check?

But uh, if you're check out after end of the semester nw lol

alisonqiu committed:
cleaned up comments in runner.py

alisonqiu committed:
Update README.md

added troubleshooting section

alisonqiu committed:
Merge branch 'zawie:master' into master

alisonqiu commented:
I opened the PR to fix the incompatibility with python3 ("ModuleNotFoundError: No module named 'commands'"). The output details are more limited when you use the updated version because I didn't fix every section of the original script, but you can still see if particular test passed or failed.

Still, I cleaned up the comments and I found that you can also use python2 instead of python3 or just python to run it, so I updated the README as well so that people can have both options. Feel free to pick which file(s) to merge:)

zawie reviewed (approved):
Nice work! 🚀
